packages/discord.js/src/structures/interfaces/InteractionResponses.js

```diff
@@ -270,6 +270,7 @@ class InteractionResponses {
   /**
    * Collects a single modal submit interaction that passes the filter.
    * The Promise will reject if the time expires.
+   * <info>Use the `showModal` method to show a modal to the user.</info>
```

Comment:
At this point in the documentation, the interaction is already responded to and thus, following this information will result in an error. This could be misleading.

Reply:
Wasn't sure how to do it properly, but I removed it now as it shouldn't be an issue anway